src/ext-test/java/org/opentripplanner/ext/flex/ScheduledDeviatedTripTest.java

```diff
@@ -238,6 +239,7 @@ private static List<Itinerary> getItineraries(
     OtpServerRequestContext serverContext
   ) {
     RouteRequest request = new RouteRequest();
+    request.journey().transit().setFilters(List.of(AllowAllTransitFilter.of()));
```

Comment:
Shouldn't this be the default somehow?

Reply:
Agree. I've changed TransitRequest class. Now default filter is List.of(AllowAllTransitFilter). I think it is OK to explicit set it in test even if it's not necessary. This way it's clear what filters are used in a test.